content/en/docs/contribute/blog/submission.md

```diff
@@ -0,0 +1,226 @@
+
+#### Article content
+
+Make sure to use second-level Markdown headings (`##` not `#`) as the topmost heading level in
+the article. The `title` you set in the front matter becomes the first-level heading for that
+page.
+
+You should follow the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/),
+but with the following exceptions:
+
+- we are OK to have authors write an article in their own writing style, so long as most readers
+  would follow the point being made
+- it is OK to use “we“ in a blog article that has multiple authors, or where the article introduction clearly indicates that the author is writing on behalf of a specific group.
+  As you'll notice from this section, although we [avoid using “we”](/docs/contribute/style/style-guide/#avoid-using-we) in our documentation,
+  it's OK to make justifiable exceptions.
+- we avoid using Kubernetes shortcodes for callouts (such as `{{</* caution */>}}`). This is
+  because callouts are aimed at documentation readers, and blog articles aren't documentation.
+- statements about the future are OK, albeit we use them with care in official announcements on
+  behalf of Kubernetes
+- code samples used in blog articles don't need to use the `{{</* code_sample */>}}` shortcode, and often
+  it is better (easier to maintain) if they do not
+
+#### Diagrams and illustrations {#illustrations}
+
+For illustrations, diagrams or charts, use the [figure shortcode](https://gohugo.io/content-management/shortcodes/#figure) can be used where feasible. You should set an `alt` attribute for accessibility.
+
+For illustrations and technical diagrams, try to use vector graphics. The blog team recommend SVG over raster (bitmap / pixel)
+diagram formats, and also recommend SVG rather than Mermaid (you can still capture the Mermaid source in a comment).
+The preference for SVG over Mermaid is because when maintainers upgrade Mermaid or make changes to diagram rendering, they may not have an easy way to contact the original blog article author to check that the changes are OK.
+
+The [diagram guide](/docs/contribute/style/diagram-guide/) is aimed at Kubernetes documentation,
+  not blog articles. It is still good to align with it but:
+- there is no need to caption diagrams as Figure 1, Figure 2, etc.
+
+The requirement for scalable (vector) imanges makes the process more difficult for
+less-familiar folks to submit articles; Kubernetes SIG Docs continues to look for ways to
+lower this bar.
+If you have ideas on how to lower the barrier, please volunteer to help out.
+
+<!-- note to maintainers of this page: vector images are easier to localize and
+     are resolution independent, so can look consistently good on different screens -->
+
+For other images (such as photos), the blog team strongly encourages use of `alt` attributes.
+It is OK to use an empty `alt` attribute if accessibility software should not mention the
+image at all, but this is a rare situation.
+
+#### Commit messages
```

Comment:
Should we add something about squashing as a best-practice? I can add a simple paragraph on it, including a link to to how to do it using git and prow, but wanted to have your take on it first.

Reply:
Added (see line 228, at time of writing).

Reply:
Looks great.